package.json

```diff
@@ -7,6 +7,7 @@
     "test": "test"
   },
   "scripts": {
+    "contracts:run": "truffle migrate --network test",
```

Comment:
I'm surprised the network used to "run the contract" (means "deploy locally") is not the "development" network but the "test" network.
Meaning that the user has to define PORT/HOST environment variables.
I can't find any mention of that env var in the README.
Did I miss something?

Reply:
Yes, you missed something. The user doesn't do not have to define any variables. This  variables are set by the `./go` script. Using the test network is consistent with how the `./go tests:unit` task works.

Reply:
OK